packit/base_git.py

```diff
@@ -317,23 +321,19 @@ def specfile_add_patches(
         if not patch_list:
             return
 
-        self.specfile.set_patches(patch_list, patch_id_digits)
+        SpecfileHelper.set_patches(self.specfile, patch_list, patch_id_digits)
 
         self.local_project.git_repo.index.write()
 
     def get_project_url_from_distgit_spec(self) -> Optional[str]:
```

Comment:
I might be wrong, but I couldn't find this being used anywhere in our projects. I would just drop it.

Reply:
https://github.com/search?q=org%3Apackit+get_project_url_from_distgit_spec&type=code



Nope, not used, can be removed